lib/rubocop/cop/rspec/it_behaves_like.rb

```diff
@@ -28,14 +29,12 @@ class ItBehavesLike < Cop
 
         def on_send(node)
           example_inclusion_offense(node, alternative_style) do
-            add_offense(node)
+            add_offense(node) do |corrector|
```

Comment:
Shouldn't there be a `message` parameter to `add_offense`?
```suggestion
            add_offense(node, message: format(MSG, replacement: style, original: alternative_style)) do |corrector|
```